DEVELOPMENT.md

```diff
@@ -4,12 +4,13 @@
 This is the simplest configuration for developers to start with.
 
 ### Initial Setup
-1. Run `docker compose run --rm django ./manage.py migrate`
-2. Run `docker compose run --rm django ./manage.py createcachetable`
-3. Run `docker compose run --rm django ./manage.py createsuperuser --email $(git config user.email)`
+1. Run `docker compose build --build-arg USERID=$(id -u) --build-arg GROUPID=$(id -g) --build-arg LOGIN=$(id -n -u)` to build the development container image. This builds the image to work with your (non-root) development user so that the linting and formatting commands work inside and outside of the container. If you prefer to build the container image so that it runs as `root`, you can omit the `--build-arg` arguments (but you will likely run into trouble running those commands).
```

Comment:
In my case following instructions failed on the server (`smaug`, note: used `docker-compose` since my docker is old there -- 20.x.x) where I had prior copy of the `minio/minio:latest` already available and docker compose did not aggressively pull them before running.


```suggestion
0. Run `docker compose pull` to ensure that possibly present local `:latest` docker images are up to date.
1. Run `docker compose build --build-arg USERID=$(id -u) --build-arg GROUPID=$(id -g) --build-arg LOGIN=$(id -n -u)` to build the development container image. This builds the image to work with your (non-root) development user so that the linting and formatting commands work inside and outside of the container. If you prefer to build the container image so that it runs as `root`, you can omit the `--build-arg` arguments (but you will likely run into trouble running those commands).
```

Reply:
Good catch, thank you. I took your suggestion, rewording a bit: 6acf8163534fc1dc5adf6ec74487804d70d8e5de

Reply:
I suppose we can specify a minimal version, but as you discovered, even your "ancient" version worked (you just had to supply a different command). What do you recommend here?